deploy/helm/sumologic/templates/setup/job.yaml

```diff
@@ -68,9 +68,16 @@ spec:
         {{ else }}
         env:
         - name: SUMOLOGIC_ACCESSID
-          value: {{ required "A valid .Values.sumologic.accessId entry required!" .Values.sumologic.accessId }}
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "sumologic.metadata.name.setup.secret" . }}
+              key: sumologic_accessid
         - name: SUMOLOGIC_ACCESSKEY
-          value: {{ required "A valid .Values.sumologic.accessKey entry required!" .Values.sumologic.accessKey }}
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "sumologic.metadata.name.setup.secret" . }}
+              key: sumologic_accesskey
+        {{ end }}
```

Comment:
We are cutting `env:` by moving this `{{end}}` for rest of environmental values

Reply:
Yeah, in the end I just changed the keys in the secret and added an unconditional `envFrom` with only the secret name changing.